src/shared/ui/widget-title/widget-title.module.scss

```diff
@@ -9,29 +9,34 @@
 }
 
 .small {
+  padding-bottom: 24px;
   font-size: 36px;
   line-height: 44px;
   letter-spacing: 0;
 }
 
 .medium {
-  font-size: 44px;
-  line-height: 52px;
-
   @include media-tablet {
+    padding-bottom: 16px;
     font-size: 36px;
     line-height: 44px;
   }
+
+  padding-bottom: 20px;
+  font-size: 44px;
+  line-height: 52px;
 }
```

Comment:
I'd prefer to have css properties listed first, and only then media queries
```
.medium {
  padding-bottom: 20px;
  font-size: 44px;
  line-height: 52px;

  @include media-tablet {
    padding-bottom: 16px;
    font-size: 36px;
    line-height: 44px;
  }
}
```
Let's discuss this with the team.